File: Ohtap E Commerce Domain Models/Development/PHP/Objects/Shippings/ShippingBasic .php
<?php
namespace Ohtap\ECommerce\DomainModels\Objects\Shippings;



class ShippingBasic implements IShippingData
{
    public $Title;
    public $Name;
    public $Email;
    public $Fax;
    public $Address;//AddressBasic
    public $Phone;
    public $Description;
    public $State;
    public $OrderNo;
    public $ID;

}

File: Ohtap E Commerce Domain Models/Development/NetFramework/Models Objects/Serializable Objects/Shippings/ShippingBasic.cs
using Ohtap.ECommerce.Domain.Models.Serializable.Objects.Addresses;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Ohtap.ECommerce.Domain.Models.Objects.Addresses;
using Ohtap.ECommerce.Domain.Models.Objects.Shippings;

namespace Ohtap.ECommerce.Domain.Models.Serializable.Objects.Shippings
{
    public class ShippingBasic : IShippingData
    {
        public string ID { get; set; }
        public string Code { get; set; }
        public string Name { get; set; }
        public int Order { get; set; }
        public string Description { get; set; }
        public int State { get; set; }
        public string Phone { get; set; }
        public string Email { get; set; }
        public string Fax { get; set; }
        public IAddressData Address { get; set; }
    }
}
